solution_guides/percona-openebs/README.md

```diff
@@ -0,0 +1,43 @@
+# pxс-operator: A chart for installing Percona Kubernetes Operator for Percona XtraDB Cluster With OpenEBS
+
+This chart implements the Percona XtraDB Cluster Operator deployment. [Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/index.html) is a database clustering solution for MySQL. The Operator itself can be found here:
+* <https://github.com/percona/percona-xtradb-cluster-operator>
+
+## Pre-requisites
+* Kubernetes 1.16+
+* PV support on the underlying infrastructure - only if you are provisioning persistent volume(s).
+* Helm v3
+
+## Deployment Details
+* <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>
+
+## Chart Details
+This chart will:
+* deploy a PXC Operator Pod for the further MySQL XtraDB Cluster creation in K8S.
+
+### Installing the Chart
+To install the chart with the `pxc` release name using a dedicated namespace (recommended):
+
+```sh
+helm repo add percona https://percona.github.io/percona-helm-charts/
```

Comment:
This readme refers to Percona only deployment. Should we be updating it to reflect OpenEBS also

Reply:
We will be taking care of this in the next PR.